### ISSUE TITLE
Files not added to list of files

### ISSUE THREAD

**Issue by sylveon** [tvrenamer/tvrenamer#99]
Windows 10.0.10586.164
Java 1.8.0_71

I can't add any file, whether it is by folder, a single file or multiple files

See the gif:
![gif](https://cloud.githubusercontent.com/assets/6440374/13854448/e7fe4f1e-ec41-11e5-958a-8629bf39210a.gif)

**Comment by charlesmilette** [comment]
More info from debug:
`[17/03/16 13:16:41,628] SEVERE UIStarter#addFiles Couldn't parse file: C:\Users\Charles\Desktop\Fullmetal Alchemist\Fullmetal Alchemist BrotherHood 02.mkv`

**Comment by vipuldelwadia** [comment]
This is because TVRenamer doesn't currently support filenames which don't have a season number in them, such as your example `Fullmetal Alchemist BrotherHood 02.mkv`.

If you rename the file to include the season number then it will be picked up in TVRenamer, for example:

- `Fullmetal Alchemist BrotherHood s01e02.mkv`
- `Fullmetal Alchemist BrotherHood 1.02.mkv`
- `Fullmetal Alchemist BrotherHood 1x02.mkv`

![](https://dl.dropboxusercontent.com/spa/qkw7gkotjvpbfog/kumr_emx.png)

**Comment by charlesmilette** [comment]
Thank you for the tip. It worked after that. But I would appreciate if the season number isn't required.

**Comment by vipuldelwadia** [comment]
Yep, I've labelled it as a future enhancement :)

**Comment by eprenamer** [comment]
I created a new Issue for the request of "recognize an episode number without a season number", #309 

I made a new issue because I think this one points at another problem.

Sometimes, with a file, we cannot find any information about the show we identified.  Or sometimes, the episode has a season/episode combination that doesn't seem to exist.  In those cases, the file still goes into the table, and we put up an informative error message.

But in the case of a file that appears not to have season information, we "quietly" ignore it.

By "quietly", I mean that in debug mode, it just gets a log message, whereas in production mode, it is completely silent.

@vipuldelwadia @daveharris Was there any particular reason for this decision?  I'd be inclined to put all movie files into the table, and then try to provide useful messages for those that could not be parsed.
